include/RAJA/pattern/nested.hpp

```diff
@@ -275,12 +275,13 @@ RAJA_INLINE void forall(const Pol &p, const SegmentTuple &st, const Body &b)
 
 }  // end namespace nested
 
+template<typename... Args>
+RAJA_HOST_DEVICE constexpr auto make_tuple(Args&&... args){
```

Comment:
auto without trailing return type requires c++14 support

you can add the trailing return type as follows (note: it's also important to perfectly forward any universal references):

```cpp
template<typename... Args>
RAJA_HOST_DEVICE constexpr auto make_tuple(Args&&... args)
-> decltype(camp::make_tuple(std::forward<Args>(args)...)) {
  return camp::make_tuple(std::forward<Args>(args)...);
}
```

Reply:
Alternatively, I believe adding `using ::camp::make_tuple;` within the RAJA namespace should enable visibility of `RAJA::make_tuple` without the need for a redefinition